Octokit.Reactive/Clients/IObservableRepositoryCommentsClient.cs

```diff
@@ -15,10 +21,21 @@ public interface IObservableRepositoryCommentsClient
         /// <param name="number">The comment id</param>
         /// <returns></returns>
         [SuppressMessage("Microsoft.Naming", "CA1716:IdentifiersShouldNotMatchKeywords", MessageId = "Get",
-            Justification = "Method makes a network request")]
+             Justification = "Method makes a network request")]
         IObservable<CommitComment> Get(string owner, string name, int number);
 
         /// <summary>
+        /// Gets a single Repository Comment by number.
+        /// </summary>
+        /// <remarks>http://developer.github.com/v3/repos/comments/#get-a-single-commit-comment</remarks>
+        /// <param name="repositoryId">The ID of the repository</param>
+        /// <param name="number">The comment id</param>
+        /// <returns></returns>
```

Comment:
Apologies, I should have been clearer - the whole `<returns></returns>` element can be removed from these docs.